pkgs/misc/screensavers/betterlockscreen/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  stdenv, makeWrapper, fetchFromGitHub, substituteAll,
+  imagemagick, i3lock-color, xdpyinfo, xrandr, bc, feh
+}:
+
+stdenv.mkDerivation rec {
+  name = "betterlockscreen-${version}";
+  version = "3.0.1";
+
+  src = fetchFromGitHub {
+    owner = "pavanjadhaw";
+    repo = "betterlockscreen";
+    rev = version;
+    sha256 = "0jc8ifb69shmd0avx6vny4m1w5dfxkkf5vnm7qcrmc8yflb0s3z6";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  patches = [ src = ./replace-i3lock.patch ];
```

Comment:
You should drop `src = ` here, it's a list of paths to patches

Reply:
Sorry that is a complete oversight from me. Should be resolved now!